registry.tcl

```diff
@@ -719,4 +719,9 @@ extension EGL_NV_stream_consumer_eglimage {
     flags       public
     filename    extensions/NV/EGL_NV_stream_consumer_eglimage.txt
 }
-# Next free extension number: 140
+extension EGL_NV_stream_consumer_eglimage {
+    number      140
```

Comment:
The extension number here and in the spec text will need to be updated when the PR is actually accepted, up through 141 are used at the moment.